input/pagecontent/StructureDefinition-dk-core-related-person-intro.md

```diff
@@ -0,0 +1,37 @@
+### Scope and usage
+The Danish Core Related Person profile is intended to encapsulate the most common and basic data model of persons related to patients/persons in Danish healthcare systems covering both primary sector, secondary sector (specialty doctors and hospitals) and municipality cases. As such the profile should be usable in most Danish contexts. Note that the profile is usable for related persons to those that receive health and social care in municipalities, even in contexts where receivers of care are more commonly known as citizens ([DA] borgere).
+
```

Comment:
We should add that this profile does and should not represent persons given power og attorney ([DA] Fuldmagt) where the Consent resource is more appropriate. The same person may though also appear as a related person with another relationship, e.g. Spouse.

Reply:
@christiangasser-lakeside foreslaget tilføjes

Reply:
Updated documentation